module/headers/vector2.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
+struct vector2 { int x; int y; };
```

Comment:
The better name for this struct is `Point`, also please use pascal case for structures